recipes-tag/collection/meta.yaml

```diff
@@ -4,7 +4,7 @@
 
 package:
   name: collection
-  version: {{ year }}-{{ cycle }}.{{ version }}
+  version: {{ year }}C{{ cycle }}.{{ version }}
```

Comment:
Is is "C" or _ better ?  Discuss ? ;-)

Reply:
Can we also drop the '20' as we are not time travelers and hopefully this will get overhauled before 2100? ;-)

Reply:
We renamed the _environments_ from 18Q1.0 to 2018-1.0 for consistency with the directory naming scheme. The only issue here in the dash, which is allowed in environment names but disallowed in version specs.

Reply:
`2018_1.0` seems more consistent with `2018-1.0`.

Reply:
... similarly to `eiger-io` package is imported as `eiger_io`. But I don't mind if we use `C`.